src/Microsoft.ML.TimeSeries/ExtensionsCatalog.cs

```diff
@@ -121,5 +121,9 @@ public static class TimeSeriesCatalog
         public static SsaSpikeEstimator DetectSpikeBySsa(this TransformsCatalog catalog, string outputColumnName, string inputColumnName, int confidence, int pvalueHistoryLength,
             int trainingWindowSize, int seasonalityWindowSize, AnomalySide side = AnomalySide.TwoSided, ErrorFunction errorFunction = ErrorFunction.SignedDifference)
             => new SsaSpikeEstimator(CatalogUtils.GetEnvironment(catalog), outputColumnName, confidence, pvalueHistoryLength, trainingWindowSize, seasonalityWindowSize, inputColumnName, side, errorFunction);
+
+        public static SrCnnAnomalyEstimator DetectAnomalyBySrCnn(this TransformsCatalog catalog, string outputColumnName, string inputColumnName,
+            int windowSize, int backAddWindowSize, int lookaheadWindowSize, int averageingWindowSize, int judgementWindowSize, double threshold)
```

Comment:
Are any of these parameters defaultable? Or are they all required all the time?

Reply:
Added default values for the parameters.